src/app/wallet/send/send.component.scss

```diff
@@ -206,3 +210,8 @@ tr {
   background-color: #a1a1a1;
   border-bottom-color: #222828;
 }
+
+.padding-5
```

Comment:
padding 5, with padding 7 top? please call it padding-top-7 or something that makes sense